examples/RNOneSignal/package.json

```diff
@@ -8,7 +8,7 @@
 	"dependencies": {
 		"react": "16.3.1",
 		"react-native": "0.55.4",
-      "react-native-onesignal": "3.2.12",
```

Comment:
Would like to keep this pointing to a real version instead of a local copy for a more real world example.
Maybe we create a new project for unit tests?

Reply:
I think the example project should always relate to the version on the repo itself. This will allow more reliable CI.